docs/versions/latest/Release.md

```diff
@@ -497,18 +533,18 @@ export class HandleNetworkConfigurationBackup {
 
 ## Improvements
 
-[//]: # (### *Improvement Name* )
-[//]: # (Talk ONLY regarding the improvement)
-[//]: # (Optional But higlhy recommended)
-[//]: # (#### Previous Behavior)
-[//]: # (Explain how it used to behave, regarding to the change)
-[//]: # (Optional But higlhy recommended)
-[//]: # (#### New Behavior)
-[//]: # (Explain how it behaves now, regarding to the change)
-[//]: # (Optional But higlhy recommended Specify *NONE* if missing)
-[//]: # (#### Relevant Documentation:)
+[//]: # "### *Improvement Name* "
+[//]: # "Talk ONLY regarding the improvement"
+[//]: # "Optional But higlhy recommended"
+[//]: # "#### Previous Behavior"
+[//]: # "Explain how it used to behave, regarding to the change"
+[//]: # "Optional But higlhy recommended"
+[//]: # "#### New Behavior"
+[//]: # "Explain how it behaves now, regarding to the change"
```

Comment:
For this kind of changes ( ) to " " and * * to _ _ could you double check with @ktsonchev , he started updating documentation for similar things and I want to double check that everything is alligned

Reply:
Discussed offline.